src/containers/treePage/index.tsx

```diff
@@ -0,0 +1,248 @@
+    circumference: 3140000,
+    coverage: 'leaves',
+    discoloring: true,
+    pooling: false,
+    light: true,
+    bicycle: true,
+    fence: 'yes',
+    treeNotes: 'I think it looks pretty good',
+    siteNotes: 'dirty',
+  }]
+};
+
+const TreePage: React.FC<SiteProps> = ({
+  address,
+  entries
+}) => {
+
+  const latestEntry: SiteEntry = entries[entries.length - 1];
+
+  const description = "Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer cronut pok pok gentrify flannel salvia deep v pork belly"
+
+  return(
+    <>
+      <Helmet>
+        <title>Tree</title>
+        <meta
+          name="Individual Tree Page"
+          content="Description of a particular tree site, allows users to adopt a tree and monitor their stewardship activities."
+        />
+      </Helmet>
+      <PageLayout>
+        <TreePageContainer> 
+          {/*Change to tree map route once that page is finished*/}
+          <ReturnButton to={Routes.HOME}> 
+              {`<`} Return to Tree Map
+          </ReturnButton> 
+          <TreeMainContainer>
+          <Row>
+            <Col span={7}>
+              <TreeInfoContainer>
+                <TreeImage
+                  src={placeholder}
+                />
+              </TreeInfoContainer>
+            </Col>
+            <Col span={10}>
+              <TreeInfoContainer>
+                { latestEntry.commonName && 
+                  <PageHeader 
+                    pageTitle={ latestEntry.commonName }
+                  />
+                }
+                <Title level={2}>
+                  { address }
+                </Title>  
+                <Paragraph>
+                  { description }
+                </Paragraph>
+                <Button type="primary" size="large">
+                  Adopt
+                </Button>  
+                <StewardshipContainer>
+                  <Title level={3}>Record Tree Care</Title>
+                  <StewardshipForm />
+                </StewardshipContainer>
+              </TreeInfoContainer>
+            </Col>
+            <Col span={7}>
+              <TreeCareContainer>   
+                <TreeCareTitle>
+                  Tree Care Activity
+                </TreeCareTitle>
+                {
+                  dummyCare.map((value: TreeCare) => {
+                    return (
+                      <CareEntry>
+                        <EntryDate>
+                          {value.date}
+                        </EntryDate>
+                        <Gap />
+                        <EntryMessage>
+                          {value.message}
+                        </EntryMessage>
+                      </CareEntry>
+                    )
+                  })
+                }
+              </TreeCareContainer>
+            </Col>
+            </Row>
+          </TreeMainContainer>
+          <Row>
+            <EntrySpace>
+              {
+                Object.entries(latestEntry).map(([key, value]) => {
```

Comment:
We shouldn't show every piece of data - I don't think we need id, username (privacy concern here), or tree present (I think we should only be displaying pages if there's a tree there)